README.md

```diff
@@ -236,46 +255,82 @@ The returned array contains objects of type `StreemCallLogEntry`, which have the
     endDate: Date?             // Session end time.
     participants: [StreemUser] // Session participants. One for an Onsite Streem, two for a two-way Streem.
     streemshotsCount: Int      // The number of Streemshots captured during the session.
+    hasMesh: Bool              // Whether the streem has a mesh with it or not.
 ```
 
 ### Displaying and Editing Streemshots
 
-Once you have fetched the call log, you may display or edit the Streemshots associated with any of the calls.
+Once you have fetched the call log, you may display or edit the artifacts associated with any of the calls.
 
-First, obtain a `StreemshotManager` for the call:
+First, obtain an `ArtifactManager` for the call. The call to `artifactManager` takes a callback which will provide the index and type of the artifact, as well as if that artifact was retrieved successfully:
```

Comment:
`a callback which will provide, for each artifact associated with the call, the type and index of the artifact, as well as whether that artifact was retrieved successfully`

Reply:
👍